docs/source/using/index.adoc

```diff
@@ -6,6 +6,7 @@ This section describes how to manage your Minishift VM and interact with your
 OpenShift cluster. It also helps troubleshooting common issues.
 
 - link:../using/managing-minishift{outfilesuffix}[Managing Minishift]
+- link:../using/proxies{outfilesuffix}[HTTP/HTTPS Proxies]
```

Comment:
"Managing Minishift" has currently the theme of the basic life cycle methods start/stop/delete plus the sections which are relevant to this major life cycles. Proxies an runtime options are really directly related to this are relevant in particular for start. Really add-ons fitted quite well there as well, but might have been a big too long. However, the current content seems to me coherent as is. 

I don't get why it would move under quickstart. A quickstart should be a short demo of the happy path for the Minshift usage with pointers into more detailed sections. Moving  start/stop/delete into quickstart adds ways to much detailed doc into it. And what's the structure here? "Basic Minshift commands" (start/stop/delete) and then "Starting Minishift", How does this fit together. I don't expect the quickstart to start with such a detailed documentation of start/stop/delete. And "Networking" gets completely misplaced in this case.